src/plays/image-collector/ImageCollector.js

```diff
@@ -0,0 +1,54 @@
+import PlayHeader from "common/playlists/PlayHeader";
+import { useState } from "react";
+import "./styles.css";
+
+function ImageCollector(props) {
+  const [urlInput, setUrlInput] = useState("");
+  const [images, setImages] = useState([]);
+  // const images = useRef([]);
```

Comment:
can you remove this comment if it's not required?

Reply:
@murtuzaalisurti removed it

Reply:
great, looks good